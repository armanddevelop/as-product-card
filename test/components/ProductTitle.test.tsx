import React from 'react';
import render from "react-test-renderer"
import ProductTitle from '../../src/components/ProductTitle';
import {ProductCard} from '../../src/components/ProductCard';
import {product1} from "../data/products";



describe('Product Title component UT', () => {
  test('Show the right component with the title', ()=>{
    const wrapper = render.create(
    <>
      <ProductCard product={product1}>
        {
          ()=>(<ProductTitle title={3} className='hola'/>)
        }
      </ProductCard>
    </>
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
  })
});
