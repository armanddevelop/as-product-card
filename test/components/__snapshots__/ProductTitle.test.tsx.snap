// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Title component UT Show the right component with the title 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription hola"
  >
    coffee Mug-Card without image
    3
  </span>
</div>
`;
